.drone.yml

```diff
@@ -22,41 +27,82 @@ steps:
         - skip-ci*
     event:
       exclude:
-      - pull_request
       - tag
   environment:
     DOCKER_PASSWORD:
       from_secret: DOCKER_PASSWORD
   commands:
     - export DOCKER_HOST=unix:///var/run/host.sock
-    - docker login -u 'robot$drone' -p $DOCKER_PASSWORD harbor.dsrd.libraries.psu.edu
-    - docker build -t harbor.dsrd.libraries.psu.edu/library/scholarsphere:$DRONE_BUILD_NUMBER . 
-    - docker tag harbor.dsrd.libraries.psu.edu/library/scholarsphere:$DRONE_BUILD_NUMBER harbor.dsrd.libraries.psu.edu/library/scholarsphere:ci
-    - docker push harbor.dsrd.libraries.psu.edu/library/scholarsphere:$DRONE_BUILD_NUMBER
+    - docker build -t harbor.dsrd.libraries.psu.edu/library/scholarsphere:$DRONE_BUILD_NUMBER --target base . 
+    - docker build -t harbor.dsrd.libraries.psu.edu/library/scholarsphere:$DRONE_BUILD_NUMBER-rspec --target rspec . 
+    - docker build -t harbor.dsrd.libraries.psu.edu/library/scholarsphere:$DRONE_BUILD_NUMBER-eslint --target eslint . 
+    - docker build -t harbor.dsrd.libraries.psu.edu/library/scholarsphere:$DRONE_BUILD_NUMBER-niftany --target niftany . 
   volumes:
   - name: docker-sock
     path: /var/run/host.sock
 
 - name: eslint
   <<: *test_common
   commands:
-    - cd /app
-    - yarn install --development
-    - yarn lint
+    - export DOCKER_HOST=unix:///var/run/host.sock
+    - docker run harbor.dsrd.libraries.psu.edu/library/scholarsphere:$DRONE_BUILD_NUMBER-eslint
 
 - name: niftany
   <<: *test_common
   commands:
-    - cd /app
-    - bundle exec niftany
+    - export DOCKER_HOST=unix:///var/run/host.sock
+    - docker run harbor.dsrd.libraries.psu.edu/library/scholarsphere:$DRONE_BUILD_NUMBER-niftany
 
 - name: test
   <<: *test_common
   commands:
-    - cd /app
-    - bundle exec rails db:create
-    - bundle exec rails db:migrate
-    - bin/ci-rspec
+    - export DOCKER_HOST=unix:///var/run/host.sock
+    - apk add bind-tools
+    - export POSTGRES_HOST=$(dig +short postgres)
+    - export MINIO_HOST=$(dig +short minio)
+    - docker run
+        --net host
+        -e POSTGRES_PASSWORD=scholarsphere
```

Comment:
should this be hidden or do we care? I'm not sure I do :)

Reply:
it's ephemerial, i don't think we care.

Reply:
since the host name is hidden this probably doesn't matter?